test/benchmarks/driverBench/index.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 
+const Promise = require('../../../lib/promise_provider').get();
```

Comment:
is it possible to use `ignorePatterns` to ignore files in `test` for this rule? I don't think it buys us anything to force these declarations in our test files, we only want it in our `lib` files to provide support to our users.

Reply:
This is the opposite of what you [mentioned earlier](https://mongodb.slack.com/archives/GGWBN4ZNK/p1588272216015300?thread_ts=1588271504.011700&cid=GGWBN4ZNK) regarding adding these to tests because it' allows us to test the driver using multiple promise implementations.

I'm not sure if it's possible for `eslint` to `ignore` a specific rule (`eslint-plugin-promise`) for a specific folder (tests). Without possibly having two different `eslint` configs.